libpod/container_internal_linux.go

```diff
@@ -282,3 +285,7 @@ func (c *Container) addNamespaceContainer(g *generate.Generator, ns LinuxNS, ctr
 
 	return nil
 }
+
+func (c *Container) readOnly() bool {
```

Comment:
I think we should make this public - move to container.go and rename to `IsReadOnly()` - seems like a useful thing

Reply:
Ok